hack/verify-pylint.sh

```diff
@@ -32,7 +33,9 @@ else
 fi
 
 export PYLINTHOME=$TEST_TMPDIR
-pylint="$(dirname $0)/pylint_bin"
 
 shopt -s extglob globstar
-${pylint} !(gubernator|external|vendor|bazel-*)/**/*.py
+
+# TODO(clarketm) there is no version of `pylint` that supports "both" PY2 and PY3
+# I am disabling pylint checks for python3 files until migration complete
+$"$DIR/pylint_bin" !(metrics|gubernator|external|vendor|bazel-*)/**/*.py
```

Comment:
Is the first `$` a typo?

Reply:
@cjwagner - for some reason I thought this was required by bash for command substitution, but I was mistaken and will fix.

Reply:
@cjwagner - [updated](89bdf78b1b7ffa5c27cfc19f8bc6a1d28b184f04)